jsbasics/018_leapyears.js

```diff
@@ -0,0 +1,12 @@
+// Question 18 =>  Write a program that prints the next 20 leap years.
+// num is defined as the present year
+
+leapYear = num => {           
+    let upto = parseInt(num) + 20;
+    for (let i = num; i <= upto; i++) {
+        if (((i % 4 == 0) && (i % 100 != 0)) || (i % 400 == 0)) {
+            console.log("Leap year : " + i);
+        }
+    }
+}
+leapYear(2020);
```

Comment:
program have no issue but here we need to print next 20 no. of leap years.this program print only leap years that only in between num and num+20.